### PR TITLE
SRCH-1278 update api docs

### PR DESCRIPTION
**Pull request by lsamuels-fearless**

### CODE REVIEW

File: app/views/sites/api_instructions/_show_web_md.html.haml [hunk 1]
```diff
@@ -1,485 +1,77 @@
 :markdown
-  <!-- Reminder to update https://open.gsa.gov/api/searchgov-results/ with any changes here. -->
-
-  This API exposes all relevant results “modules” in a single JSON call, including:
-
-
+  ## Overview
 
+  The Search.gov Results API exposes all relevant results “modules” in a single JSON call, including the following:
```
Comment (jmax-fearless) [inline app/views/sites/api_instructions/_show_web_md.html.haml:4]: Maybe just a nit, but I'm not a fan of emphasis quotes. What are you trying to convey with them, and is there a better way to do it? Maybe a link somewhere, if modules is a term with specific meaning you want to emphasize here, or just _underline_ or *star* emhasis.

File: app/views/sites/api_instructions/_show_web_md.html.haml [hunk 2]
```diff
@@ -1,485 +1,77 @@
...
-  ## Note about Web Results and Endpoints
-
+  ## Getting started
 
+  Please refer to the detailed documentation for the Results API, now publicly available through the [OpenGSA Portal](https://open.gsa.gov/api/searchgov-results/). The information below serves as an introduction.
 
-
-  The endpoint you use to retrieve web results through this API will depend on the method we used to index your content. If we don't yet have your content indexed, you won't see results in the API.
-  
   We can index content using your [XML sitemap](https://search.gov/blog/sitemaps.html) (preferred), [RSS feeds](#{site_rss_feeds_path(@site)}), or the [indexing API, i14y](https://search.gov/developer/i14y.html). We can also deploy a crawler on a limited basis.
```
Comment (jmax-fearless) [inline app/views/sites/api_instructions/_show_web_md.html.haml:19]: Much better than the original. More concise while still retaining all of the information.

File: app/views/sites/api_instructions/_show_web_md.html.haml [hunk 3]
```diff
@@ -1,485 +1,77 @@
...
-
 
+  Sites indexed via sitemaps or crawling will use the `/i14y` endpoint. Because most users are in this category, the example API calls in the documentation are to this endpoint. Sites indexed via RSS will use the top level `/` endpoint, please modify your calls accordingly.
 
-  ## Required Parameters
 
+  ## API Key and Resource URL
 
+  The endpoint is `#{api_scheme_and_host}/api/v2/search/i14y`.
+  `#{api_scheme_and_host}/api/v2/search/i14y?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query=SEARCH_TERM_ENTERED_IN_YOUR_SEARCH_BOX`
```
Comment (jmax-fearless) [inline app/views/sites/api_instructions/_show_web_md.html.haml:28]: Anybody using this _ought_ to know, but it wouldn't hurt to emphasize that the search term needs to be URL-encoded.

File: app/views/sites/api_instructions/_show_web_md.html.haml [hunk 4]
```diff
@@ -1,485 +1,77 @@
...
-
-  * ### web:spelling_correction
-
-
-
-
-    Spelling correction for your search term.
-
-
+  For descriptions of the following parameters, please see the full documentation on the [OpenGSA Portal](https://open.gsa.gov/api/searchgov-results/).
```
Comment (jmax-fearless) [inline app/views/sites/api_instructions/_show_web_md.html.haml:35]: I'm presuming that the preceding section got transferred bodily to the OpenGSA portal page referred to; it certainly ought to still be around _somewhere_.

File: app/views/sites/api_instructions/_show_web_md.html.haml [hunk 5]
```diff
@@ -1,485 +1,77 @@
...
-    | thumbnail:width        | Thumbnail pixel width
-    | thumbnail:height       | Thumbnail pixel height
-
-
-
+  For instance, if you set the query `example` to route to `https://www.search.gov`, then the expected response will be the following:
 
+  `{"route_to":"https://www.search.gov/"}`
 
+  To fully implement this, you will have to add logic that will process this response and redirect the user. A site-specific API call and javascript sample are available within the  Search Results API Instructions.
```
Comment (jmax-fearless) [inline app/views/sites/api_instructions/_show_web_md.html.haml:77]: Maybe make the 'Search Results API Instructions' at the end of this line a link?

**Comment by lsamuels-fearless** [comment]
@jmax-fearless thanks for the review. I didn't write the copy myself, but @dawnpm or @ayushchak may be interested in reviewing these suggestions.

**Comment by ayushchak** [comment]
@lsamuels-fearless - I added a new version of the txt doc to the JIRA ticket. If it's easier to just merge the PR, that's fine as well. Thanks!

**Pull request merged**